lib/mongo/protocol/msg.rb

```diff
@@ -199,7 +199,7 @@ def documents
       # auto-encryption options, the client has not been instructed to bypass
       # auto-encryption, and mongocryptd determines that this message is
       # eligible for encryption. A message is eligible for encryption if it
-      # represents one of the command types whitelisted by libmongocrypt and it
+      # represents one of the command types allowlisted by libmongocrypt and it
```

Comment:
I think these need to be hyphenated, i.e. `allow-listed`. Looking through the rest of the diff, people use the phrase "allow list" as two words, plus "allowlist" doesn't flow very well.